Framework/PythonInterface/mantid/plots/plotfunctions.py

```diff
@@ -8,28 +8,31 @@
 #
 #
 from __future__ import (absolute_import, division, print_function)
+
+from mantid.plots.utility import MantidAxType
 import collections
 import sys
 
-import numpy
-from skimage.transform import resize
-import mantid.kernel
-import mantid.api
-from mantid.plots.helperfunctions import *
 import matplotlib
 import matplotlib.collections as mcoll
 import matplotlib.colors
 import matplotlib.dates as mdates
 import matplotlib.image as mimage
+import numpy
+from skimage.transform import resize
 
+import mantid.api
+import mantid.kernel
+from mantid.plots.helperfunctions import get_axes_labels, get_bins, get_data_uneven_flag, get_distribution, \
+    get_matrix_2d_data, get_md_data1d, get_md_data2d_bin_bounds, get_md_data2d_bin_centers, get_normalization, \
```

Comment:
A lot of good tests for the helperfunctions.py.

I think the import of get_matrix_2d_data is unused. From my point of view, the correctness of the imports from helperfunctions is important to show where tests can be potentially added, i.e. as sort of documentation.

Reply:
I can't see any unused imports (`get_matrix_2d_data` is used a few times in the [file](https://github.com/mantidproject/mantid/blob/51e034ee115331d2015bd6efd70c4ee318e3463a/Framework/PythonInterface/mantid/plots/plotfunctions.py#L230)).